forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java

```diff
@@ -312,6 +313,10 @@ public void resolve(final SpellAbility sa) {
                 tgtSA = CardFactoryUtil.abilityMorphDown(tgtCard.getCurrentState(), false);
             } else {
                 // only one mode can be used
+                if (sa.hasParam("CastTransformed")) {
+                    tgtCard.changeToState(CardStateName.Transformed);
+                }
+
```

Comment:
does it really work for Cards that are not Permanents on the Backside? or isn't the SpellAbility wrong?

Reply:
I think if the changeToState fails we can just return early.